deploy/kube-config/influxdb/heapster-controller.json

```diff
@@ -26,7 +26,7 @@
 	    "spec": {
 		"containers": [
 		    {
-			"image": "kubernetes/heapster:v0.18.1",
+			"image": "thuctnguyen/heapster:v0.18.2-rc1",
```

Comment:
I will set up a nightly build for heapster soon. Until then, can you revert this change? 
I will also make a heapster v0.19.0 release soon, that you can use with influxDB v0.9.4.